server/requirements.txt

```diff
@@ -1,19 +1,23 @@
+atomicwrites==1.2.1
```

Comment:
I can't tell where this and some of the other new requirements in here are coming from. Are they second-degree dependencies?

With the way we're using requirements.txt now, we're keeping the dependency list down to just those that are directly imported by the codebase. (We should add a set of locked dependencies, too, but haven't done that as of now)

Reply:
I am not sure where they came from either. I used PyCharm in Windows and Bash in Ubuntu. When tests passed in both env, I recreated the requirements.txt files.